en/complete_vehicles/px4_vision_kit.md

```diff
@@ -210,7 +211,7 @@ To flash the USB image to the *UP Core*:
 ### Boot the Companion Computer {#boot_mission_computer}
 
 First insert the provided USB2.0 stick into the *UP Core* port labeled `USB1`, and then power the vehicle using a 4S battery.
-The avoidance system should start within about 2.5 minutes.
+The avoidance system should start within about 1 minute (though this does depend on the USB stick supplied).
```

Comment:
Was this number from your experience? Could you check if your usb-stick as usb-2.0?

Reply:
@Jaeyoung-Lim Yes. This was the time from power until the lasery thingy showed on the camera. It is a USB2.0 - SDCZ50-032G (Cruzer Blade 32GB).

This is the stick that came with the vehicle.

Reply:
This otherwise OK?